demos/serverless/src/handlers.js

```diff
@@ -208,3 +209,55 @@ exports.end = async(event, context, callback) => {
   }).promise();
   callback(null, response);
 };
+
+exports.logs = async (event, context) => {
+  var sequenceToken = "";
+  var response = {
+    "statusCode": 200,
+    "headers": {},
+    "body": '',
+    "isBase64Encoded": false
+  };
+  try {
+    var body = JSON.parse(event.body);
+    if (!body.logs || !body.meetingId || !body.attendeeId || !body.appName) {
+      response.body = "Empty Parameters Received";
+      return response;
+    }
+    const logStreamName = "ChimeSDKMeeting_" + body.meetingId.toString();
+    var cloudWatchClient = new AWS.CloudWatchLogs({
+      apiVersion: '2014-03-28'
+    });
+    var putLogEventsInput = {
+      "logGroupName": logGroupName,
+      "logStreamName": logStreamName
+    };
+    var describeLogStreamsParams = {
+      "logGroupName": logGroupName,
+      "logStreamNamePrefix": logStreamName
+    };
+    var describeLogStreamsResponse = await cloudWatchClient.describeLogStreams(describeLogStreamsParams).promise();
```

Comment:
There's too much business logic in this function. Extract a helper - `ensureCloudWatchLogStream` which creates the stream if it doesn't exists.